Dockerfile

```diff
@@ -31,7 +33,14 @@ RUN mv $REMOTE_SOURCE_DIR/app /workspace
 WORKDIR /workspace/app
 
 # Build
-RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -a -ldflags="-X '${GO_MODULE}/version.BuildTimestamp=`date '+%Y-%m-%dT%H:%M:%S'`'" -o /workspace/manager main.go
+# the GOARCH has not a default value to allow the binary be built according to the host where the command
+# was called. For example, if we call make docker-build in a local env which has the Apple Silicon M1 SO
+# the docker BUILDPLATFORM arg will be linux/arm64 when for Apple x86 it will be linux/amd64. Therefore,
+# by leaving it empty we can ensure that the container and binary shipped on it will have the same platform.
+RUN CGO_ENABLED=0 GOOS=${TARGETOS:-linux} GOARCH=${TARGETARCH} go build -a \
+    -ldflags="-X '${GO_MODULE}/version.CommitHash=`git rev-parse --short HEAD`' \
```

Comment:
`version.CommitHash` was removed, see https://github.com/artemiscloud/activemq-artemis-operator/commit/2016476d027dd62ff2e25d8caec4a3b946d49074

Reply:
yeah, missed that. thanks! will fix it.